<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryDialog</class>
 <widget class="QDialog" name="QueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_data">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>991</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_doQuery">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Do Query</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_queries">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_queryDescription">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_query">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>991</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Query:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
